bigtop-packages/src/common/alluxio/install_alluxio.sh

```diff
@@ -65,12 +66,15 @@ while true ; do
         --bin-dir)
         BIN_DIR=$2 ; shift 2
         ;;
-        --var-dir)
-        VAR_DIR=$2 ; shift 2
+        --var-alluxio-dir)
+        VAR_ALLUXIO_DIR=$2 ; shift 2
         ;;
         --data-dir)
         DATA_DIR=$2 ; shift 2
         ;;
+		--conf-dist-dir)
+		CONF_DIST_DIR=$2 ; shift 2
+		;;
```

Comment:
should be indented with whitespace instead of tab.

Reply:
fixed